lib/order.rb

```diff
@@ -0,0 +1,26 @@
+class Order 
+  attr_reader :basket
+
+  def initialize
+    @total = 0
+    @menu = Menu.new.dishes
+    @basket = []
+  end
+
+  def choose_dish(dish)
```

Comment:
It might need to include a way ask for more than one of each dish. A hash for @basket could help on that